cfme/tests/control/test_alerts.py

```diff
@@ -187,16 +187,14 @@ def vm_crud(provider, vm_name, full_template):
 
 # TODO Replace this with the appliance fixture once complete
 @pytest.yield_fixture(scope="module")
-def snmp():
-    ssh = store.current_appliance.ssh_client
-    ssh.run_command("echo 'disableAuthorization yes' >> /etc/snmp/snmptrapd.conf")
-    ssh.run_command("systemctl start snmptrapd.service")
+def snmp(ssh_client_modscope):
```

Comment:
**REQUIRED:** ```ssh_client_modscope``` doesn't exist anymore, and should be replaced by using the ```appliance``` fixture and then ```appliance.ssh_client``` in the fixture body.